cloud/team/team.go

```diff
@@ -0,0 +1,708 @@
+		Padding:        []int{10, 50, 50, 10, 10, 10, 10},
+		DynamicPadding: true,
+		Header:         []string{"ID", "Role", "Name", "Description", "CREATE DATE"},
+	}
+	teams, err := GetWorkspaceTeams(client, workspace, teamPagnationLimit)
+	if err != nil {
+		return err
+	}
+
+	for i := range teams {
+		var teamRole string
+		for _, role := range *teams[i].Roles {
+			if role.EntityType == "WORKSPACE" && role.EntityId == workspace {
+				teamRole = role.Role
+			}
+		}
+		table.AddRow([]string{
+			teams[i].Id,
+			teamRole,
+			teams[i].Name,
+			*teams[i].Description,
+			teams[i].CreatedAt.Format(time.RFC3339),
+		}, false)
+	}
+
+	table.Print(out)
+	return nil
+}
+
+func AddWorkspaceTeam(id, role, workspace string, out io.Writer, client astrocore.CoreClient) error {
+	err := user.IsWorkspaceRoleValid(role)
+	if err != nil {
+		return err
+	}
+	ctx, err := context.GetCurrentContext()
+	if err != nil {
+		return err
+	}
+	if ctx.OrganizationShortName == "" {
+		return ErrNoShortName
+	}
+	if workspace == "" {
+		workspace = ctx.Workspace
+	}
+
+	// Get all org teams. Setting limit to 1000 for now
+	teams, err := GetOrgTeams(client)
+	if err != nil {
+		return err
+	}
+	if len(teams) == 0 {
+		return ErrNoTeamsFoundInOrg
+	}
+	teamID, err := getTeamID(id, teams)
+	if err != nil {
+		return err
+	}
+
+	mutateUserInput := astrocore.MutateWorkspaceTeamRoleRequest{
+		Role: role,
+	}
+	resp, err := client.MutateWorkspaceTeamRoleWithResponse(httpContext.Background(), ctx.OrganizationShortName, workspace, teamID, mutateUserInput)
+	if err != nil {
+		return err
+	}
+	err = astrocore.NormalizeAPIError(resp.HTTPResponse, resp.Body)
+	if err != nil {
+		return err
+	}
+	fmt.Fprintf(out, "The team %s was successfully added to the workspace with the role %s\n", teamID, role)
+	return nil
+}
+
+// Returns a list of all of an organizations teams
+func GetOrgTeams(client astrocore.CoreClient) ([]astrocore.Team, error) {
+	offset := 0
+	var teams []astrocore.Team
+	ctx, err := context.GetCurrentContext()
+	if err != nil {
+		return nil, err
+	}
+	if ctx.OrganizationShortName == "" {
+		return nil, ErrNoShortName
+	}
+
+	for {
+		resp, err := client.ListOrganizationTeamsWithResponse(httpContext.Background(), ctx.OrganizationShortName, &astrocore.ListOrganizationTeamsParams{
+			Offset: &offset,
+			Limit:  &teamPagnationLimit,
+		})
+		if err != nil {
+			return nil, err
+		}
+		err = astrocore.NormalizeAPIError(resp.HTTPResponse, resp.Body)
+		if err != nil {
+			return nil, err
+		}
+		teams = append(teams, resp.JSON200.Teams...)
+
+		if resp.JSON200.TotalCount <= offset {
+			break
+		}
+
+		offset += teamPagnationLimit
+	}
+
+	return teams, nil
+}
+
+// Prints a list of all of an organizations users
+func ListOrgTeams(out io.Writer, client astrocore.CoreClient) error {
+	table := printutil.Table{
+		Padding:        []int{10, 50, 50, 10, 10, 10},
+		DynamicPadding: true,
+		Header:         []string{"ID", "Name", "Description", "CREATE DATE"},
```

Comment:
```suggestion
		Header:         []string{"ID", "Name", "Description", "Create Date"},
```
nit: keeping headers consistent